### ISSUE TITLE
Lost network connection[BUG]

### ISSUE THREAD

**Issue by n3xus1** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#772]
**Describe the bug**
After running latest software I lost network access 
Unable to connect to local wifi AP
I am able to connect to device AP and all settings appear to be saved, when i try and save WiFi SSID and password and restart ESP, device never connects

I solved my problem as one of the three wireless AP's I have went off line.
what concerns me is all AP's have the same SSID and even after rebooting the device I wasn't able to connect to my local network. I find this very strange and this not normal but could be a design limitation. Could you please confirm

Hardware (please complete the following information):

    Pump model S100104
    6 wires
    PCB type D1 Mini

Software (please complete the following information):

    Version: 2024-07-19-0816

**Comment by visualapproach** [comment]
I don't know if "fast roaming" works with the esp. Try turning it off if you have it set.

**Comment by n3xus1** [comment]
I don't believe this is a fast roaming issue as the AP don't have an option and from what I can tell don't support it.
I think what is happening is the AP's are on different channels and the ESP is hanging on to the one AP even it is rebooted.
I did some googling and it appears a lot of people have similar issues with this and fast roaming and some have gotten around it by adding some code to search and select the AP with the same SSID with the highest signal strength then connect.
I looked at the code base but I'm really rusty as i have not coded much in the last 15 years and most of my micro stuff was 20+ years ago.
